mm2src/mm2_main/src/lp_swap/maker_swap.rs

```diff
@@ -471,6 +487,13 @@ impl MakerSwap {
     }
 
     async fn start(&self) -> Result<(Option<MakerSwapCommand>, Vec<MakerSwapEvent>), String> {
+        #[cfg(feature = "run-docker-tests")]
+        if let Ok(env_pubkey) = std::env::var("TEST_DEX_FEE_ADDR_RAW_PUBKEY") {
+            unsafe {
+                TEST_DEX_FEE_ADDR_RAW_PUBKEY = Some(hex::decode(env_pubkey).expect("valid hex"));
+            }
+        }
```

Comment:
I think, this var is needed because in docker swap tests we start new taker and maker mm2 processes and in couple tests I need to run taker with a test dex pubkey. I guess this is okay as there are other cases in the codebase using env vars to configure tests.
(I considered creating a feature for that but then we would need an extra cargo test call to run such tests in CI)

Reply:
Ofc I'd like to get rid of TEST_DEX_FEE_ADDR_RAW_PUBKEY static var too but for that I'd have to change dex_pubkey() return type (it's static ref now), did not like to do it just for tests sake